crates/turborepo-lib/src/shim/local_turbo_state.rs

```diff
@@ -185,37 +185,39 @@ impl Default for YarnRc {
 }
 
 pub fn turbo_version_has_shim(version: &str) -> bool {
-    let version = Version::parse(version).unwrap();
-    // only need to check major and minor (this will include canaries)
-    if version.major == 1 {
-        return version.minor >= 7;
+    if let Ok(version) = Version::parse(version) {
+        // only need to check major and minor (this will include canaries)
+        if version.major == 1 {
+            return version.minor >= 7;
+        }
+        version.major > 1
+    } else {
+        // In the case that we don't get passed a valid semver we should avoid a panic.
+        // We shouldn't hit this we introduce back infering package version from schema
+        // or package.json.
+        true
```

Comment:
This is technically too loose, but the correct answer for the past 21 months of `turbo` releases.